pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -843,6 +843,18 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 					Description:  `Mode for parsing entities in the message text. Default is 'HTML'`,
 					PropertyName: "parse_mode",
 				},
+				{
+					Label:        "Disable Web Page Preview",
+					Description:  "Sends the message without showing the web page preview.",
```

Comment:
We can use the descriptions from Telegram's [docs](https://core.telegram.org/bots/api):
```suggestion
					Description:  "Disables link previews for links in this message",
```